src/app/container/add-tag/add-tag.component.html

```diff
@@ -85,7 +85,7 @@ <h4 class="modal-title">Add Version Tag</h4>
             </label>
             <div class="col-sm-9 col-md-9 col-lg-9 test_parameter_div">
               <div class="input-group">
-                <input type="text" #model1="ngModel" class="form-control" name="cwlTestParameterFilePath" [(ngModel)]="unsavedTestCWLFile" minlength="3" maxlength="128" [pattern]="validationPatterns.testFilePath" title="Relative path to a WDL Test Parameter File in the Git repository." placeholder="e.g. /test.wdl.json" [disabled]="!editMode" ng-class="editMode ? 'test_parameter_input' : ''" />
+                <input type="text" #model1="ngModel" class="form-control" name="cwlTestParameterFilePath" [(ngModel)]="unsavedTestCWLFile" minlength="3" maxlength="128" [pattern]="validationPatterns.testFilePath" title="Relative path to a CWL Test Parameter File in the Git repository." placeholder="e.g. /test.cwl.json" [disabled]="!editMode" ng-class="editMode ? 'test_parameter_input' : ''" />
```

Comment:
A Tooltip object was created a while to contain all the tooltips that may be repeated.  This one seems like it should be in there.